apps/web/app/platform/study-groups/view.tsx

```diff
@@ -1,50 +1,48 @@
 "use client";
-
-import {
-    Card,
-    CardContent,
-    CardDescription,
-    CardFooter,
-    CardHeader,
-    CardTitle,
-} from "@ui/components/ui/card"
-
-import { Button } from "@ui/components/ui/button"
-import { Badge } from "@ui/components/ui/badge"
-import Link from 'next/link'
-
-
+import { Button } from "@ui/components/ui/button";
 import {
-    CameraIcon,
-    ChatBubbleIcon,
-} from "@radix-ui/react-icons";
-
-import { Mic } from "lucide-react";
+  Dialog,
+  DialogContent,
+  DialogDescription,
+  DialogFooter,
+  DialogHeader,
+  DialogTitle,
+  DialogTrigger,
+} from "@ui/components/ui/dialog";
+import { Input } from "@ui/components/ui/input";
+import { Label } from "@ui/components/ui/label";
+import { Plus } from "lucide-react";
 
 export default function View() {
-    return (
-        <>
-            <main>
-                <Card>
-                    <CardHeader>
-                        <div className="flex justify-between items-center w-full">
-                            <div className="flex gap-2">
-                                <CameraIcon className="w-5 h-5" />
-                                <ChatBubbleIcon className="w-5 h-5" />
-                                <Mic className="w-5 h-5" />
-                            </div>
-                            <Badge className="bg-green-500">Live</Badge>
-                        </div>
-                        <CardTitle className="text-2xl">MTH 242 Study Session</CardTitle>
-                        <CardDescription>Join the MTH 242 study session</CardDescription>
-                        <Link href="study-groups/session">
-                            <Button>Join</Button>
-                        </Link>
-                    </CardHeader>
-                </Card>
-            </main>
-        </>
-    );
+  return (
+    <Dialog>
+      <DialogTrigger asChild>
+        <Button variant="outline" className="h-full text-xl">
+          <Plus className="mr-2" />
+          Add Course
+        </Button>
+      </DialogTrigger>
+      <DialogContent className="sm:max-w-[450px]">
+        <DialogHeader>
+          <DialogTitle>Add Course</DialogTitle>
+          <DialogDescription>Add your course here.</DialogDescription>
+        </DialogHeader>
+        <div className="grid gap-4 py-4">
+          <div className="grid grid-cols-4 items-center gap-4">
+            <Label htmlFor="name" className="text-right">
+              Course Code
+            </Label>
+            <Input
+              id="name"
+              placeholder="ex) CS 161"
+              className="col-span-3 placeholder-opacity-50"
+            />
```

Comment:
I'd look into [react-hook-form](https://ui.shadcn.com/docs/components/form) for the course creation forms!